source/_integrations/broadlink.markdown

```diff
@@ -10,17 +11,19 @@ ha_iot_class: Local Polling
 ha_codeowners:
   - '@danielhiversen'
   - '@felipediel'
+  - '@majuss'
 ha_domain: broadlink
 ha_config_flow: true
 ---
 
 The Broadlink integration allows you to control and monitor Broadlink universal remotes, smart plugs, power strips, switches and sensors. The following devices are supported:
 
+- Wi-Fi Controlled Thermostats: `Hysen heating controller`
```

Comment:
There are many models and manufacturers for this PID: Beok TDS23WIFI-EP, Hysen HY02B05, Floureon HY03WE (probably manufactured by Hysen and sold under Floureon brand) etc.

We should add only the models that are supported here, because if we put the brand "Hysen" and some models are not supported, users will complain that the integration is not working for their devices.